ci/conan/all/conanfile.py

```diff
@@ -328,7 +328,7 @@ def _with_boost(self, required=False):
             return bool(self.options.with_boost)
 
     def _with_thrift(self, required=False):
-        # No self.options.with_thift exists
+        # No self.options.with_thrift exists
```

Comment:
Could you report this to the upstream?
https://github.com/conan-io/conan-center-index/blob/master/recipes/arrow/all/conanfile.py

Reply:
I've added the repository to my to-do list, but I'm already a number of levels upstream. I can try to do it once I clear out the rest of these splits... 


(I did Google FlatBuffers)